src/chapters/arrays/working-with-arrays.rst

```diff
@@ -0,0 +1,73 @@
+Working With Arrays
+===================
+
+Bracket Notation
+----------------
+Arrays are ordered collections, meaning they will keep the order of items they contain. That order is refered to as the index.
+Like with strings, use bracket notation and an index to access a specific item in an array ``array[index]``.
+Also like strings, an arrays index is zero based. An array's index will go from ``0`` to ``array.length - 1``.
+
+.. admonition:: Example
+
+   Print out the length of two arrays.
```

Comment:
This is not a ``length`` example. 
How about, "Print out several items from an array."